backend/app/services/attachment/mysql_storage.py

```
@@ -212,10 +212,20 @@ def _extract_attachment_id(self, key: str) -> int:
             StorageError: If key format is invalid
         """
         try:
-            # Key format: attachments/{attachment_id}
+            # Key format: attachments/{uuid}_{timestamp}_{user_id}_{attachment_id}
             parts = key.split("/")
             if len(parts) != 2 or parts[0] != "attachments":
                 raise ValueError("Invalid key format")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)